.github/workflows/deploy-branch.yml

```diff
@@ -2,6 +2,29 @@ name: Deploy to dev server
 
 on:
   pull_request:
+
+# Cancel workflow runs on PRs when the PR is updated with a newer commit.
+# Such runs will have a concurrency group named
+# `{github.workflow}-{github.ref}`,
```

Comment:
```suggestion
# `{github.workflow}/{github.ref}`,
```

Reply:
I agree, but we already have it this way in tarantool/tarantool and tarantool/sdk. Anyway, these labels are only used by the CI.